src/test/java/org/dinosaur/foodbowl/domain/member/persistence/MemberRepositoryTest.java

```diff
@@ -31,17 +33,57 @@ class MemberRepositoryTest extends PersistenceTest {
 
     @Test
     void 아이디와_일치하는_회원_조회() {
-        Member member = Member.builder()
-                .email("foodBowl@gmail.com")
-                .socialId("foodBowlId")
-                .socialType(SocialType.APPLE)
-                .nickname("foodbowl")
-                .introduction("안녕하세요")
-                .build();
-        Member saveMember = memberRepository.save(member);
+        Member saveMember = memberTestPersister.memberBuilder().save();
 
         Member findMember = memberRepository.findById(saveMember.getId()).get();
 
         assertThat(findMember.getId()).isEqualTo(saveMember.getId());
     }
+
+    @Nested
+    class 소셜아이디와_소셜타입으로_조회 {
+
+        @Test
+        void 일치하는_회원이_존재하면_회원을_조회한다() {
+            SocialType socialType = SocialType.APPLE;
+            String socialId = "abcd";
+            memberTestPersister.memberBuilder()
+                    .socialType(socialType)
+                    .socialId(socialId)
+                    .save();
+
+            Optional<Member> findMember = memberRepository.findBySocialTypeAndSocialId(socialType, socialId);
+
+            assertThat(findMember).isNotEmpty();
```

Comment:
`isPresent`로 한번에 assertThat으로 검증해도 되겠네요 !

Reply:
이 부분 잘 이해하지 못했는데 코드로 다시 한번 설명부탁드려도 될까요 🤔

Reply:
```suggestion
           assertThat(memberRepository.findBySocialTypeAndSocialId(socialType, socialId)).isPresent();
```

이 뜻이었습니다 !!

크게 중요한 부분은 아니라서 ㅎㅎ 넘어가도 괜찮아요 ~